### PR TITLE
chore: fix typo in blob.ts

### PR DESCRIPTION
**Pull request by eltociear**
attemps -> attempt

### CODE REVIEW

**Review by DanielJGeiger — APPROVED** [review]
@dwelle @ad1992 #5664 can be directly merged -- no code is changed, just a one-letter fix in two comments.

File: src/data/blob.ts [hunk 1]
```diff
@@ -356,7 +356,7 @@ export const getFileHandle = async (
 };
 
 /**
- * attemps to detect if a buffer is a valid image by checking its leading bytes
+ * attempt to detect if a buffer is a valid image by checking its leading bytes
```
Comment (dwelle) [inline src/data/blob.ts:359]:
```suggestion
 * attempts to detect if a buffer is a valid image by checking its leading bytes
```

File: src/data/blob.ts [hunk 2]
```diff
@@ -396,7 +396,7 @@ export const createFile = (
   });
 };
 
-/** attemps to detect correct mimeType if none is set, or if an image
+/** attempt to detect correct mimeType if none is set, or if an image
```
Comment (dwelle) [inline src/data/blob.ts:399]:
```suggestion
/** attempts to detect correct mimeType if none is set, or if an image
```

**Review by dwelle — APPROVED** [review]

**Comment by dwelle** [comment]
thanks @eltociear (made 3rd tense to align with the styleguide), and @DanielJGeiger

**Pull request merged**